src/internal/views/nunjucks/billing/batch-invoice.njk

```diff
@@ -72,18 +72,85 @@ invoiceDebitsTotal %}
+          Reissued <span class="screen-reader-only">bill</span>
+        {% endif %}
+
+        {% if inv.rebillingStateLabel == 'original' %}
+          Original <span class="screen-reader-only">bill</span>
+        {% endif %}
+
+        {% if inv.rebillingStateLabel == 'reversal' and inv.originalInvoiceId != null %}
+          Reversed <span class="screen-reader-only">bill</span>
+        {% endif %}
+      {% endmacro %}
+
+      {% macro displayInvoiceNumber(inv) %}
+        {% if inv.originalInvoiceId != null %}
+          <a href="/billing/batch/{{inv.billingBatchId}}/invoice/{{inv.id}}">
+            {{inv.invoiceNumber}}
+          </a>
+        {% else %}
+          <!-- original bill has different batchid -->
+          <a class="govuk-link" href="/billing/batch/{{inv.billingBatchId}}/invoice/{{inv.id}}">
+            {{inv.invoiceNumber}}
+          </a>
+        {% endif %}
+      {% endmacro %}
+
+      {% set htmlBillsLinked %}
+      <h2 class="govuk-heading-m">This bill is linked to a reissue</h2>
+      <dl class="meta govuk-!-margin-bottom-3">
+        <div class="meta__row">
+          <dt class="meta__key">Date created</dt>
+          <dd class="meta__value">{{batch.dateCreated | date }}</dd>
+        </div>
+        <!-- curently displayed bill -->
+        <div class="meta__row">
+          <dt class="meta__key">
+            {{ displayBillingState(invoice, invoice.batch.billingBatchId)}}
+          </dt>
+          <dd class="meta__value">
+            {{ invoice.invoiceNumber }}, this bill
+          </dd>
+        </div>
+        {% for linkedInvoice in invoice.linkedInvoices %}
+          {% if linkedInvoice.invoiceNumber != null %}
+          <div class="meta__row">
+            <dt class="meta__key">
+              {{ displayBillingState(linkedInvoice, invoice.batch.billingBatchId) }}
+            </dt>
+            <dd class="meta__value">
+              {{ displayInvoiceNumber(linkedInvoice) }}
+            </dd>
+          </div>
+          {% endif %}
+        {% endfor %}
+      </dl>
+      {% endset %}
+
       <div class="govuk-!-margin-bottom-9">
         {% set html %}
-
-        <p class="govuk-!-font-weight-bold govuk-!-margin-0">
-          <a href="{{ links.billingAccount }}">Billing account {{ invoice.invoiceAccount.accountNumber }}</a>
-        </p>
-        {{ invoiceAddress(invoice, invoice.invoiceAccount.company) }}
+          <p class="govuk-!-font-weight-bold govuk-!-margin-0">
+            <a href="{{ links.billingAccount }}">Billing account {{ invoice.invoiceAccount.accountNumber }}</a>
+          </p>
+          {{ invoiceAddress(invoice, invoice.invoiceAccount.company) }}
         {% endset %}
         {{ govukDetails({
           summaryText: "Billing account details",
           html: html })
         }}
+
+        {% if batch.status == 'sent' %}
+          {% if (invoice.linkedInvoices.length) or (invoice.rebillingState == 'rebilled') %}
```

Comment:
I think this or `(invoice.rebillingState == 'rebilled')` can be removed from the if statement. 
If there are linked invoices then they will always relate to rebilling.